src/wesnoth.cpp

```diff
@@ -1043,32 +1043,24 @@ int main(int argc, char** argv)
 	auto args = read_argv(argc, argv);
 	assert(!args.empty());
 
+	bool log_redirect = true, terminal_implied = false;
+	// This is optional<bool> instead of tribool because value_or() is exactly the required semantic
+	std::optional<bool> terminal_force;
```

Comment:
These things are complex enough that having documentation for them would be good, even though they're local variables.

Reply:
I started writing some and realized that I'm not even sure what `log_redirect` is for. It's only used in Windows.

Reply:
I see `log_redirect` still exists, but it looks as if `terminal_implied`, `file_force` and `terminal_force` ought to cover all the possibilities. What is the remaining purpose of `log_redirect`?

Reply:
Mostly I was just trying to be careful about touching the Windows-specific side of this because I wasn't quite sure what it was all for. You might be right – perhaps `log_redirect` (and `--wnoredirect`) is now obsolete. I'm not quite sure, so I'd prefer to get confirmation from someone who actually uses Windows.